spec/v1/server.md

```
@@ -373,51 +372,42 @@ that are signed in to the same account. It is a third use of the
-`https://<host>/pair?mode=join&code=XXXX-XXXX`) and on screen for typed
-entry. The `user_code` alone is insufficient to access slot state without
-a same-user session, but it is the lookup key for `/challenge`, `/claim`,
-and `/approve`.
+`https://<host>/pair?code=XXXX-XXXX`) and on screen for typed entry. The
+`user_code` alone is insufficient to access slot state without a
+same-user session, but it is the lookup key for `/challenge` and
+`/approve`.
 
 **Slot lifecycle.** State machine:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)